kademlia/network.py

```diff
@@ -48,6 +48,9 @@ def stop(self):
         if self.transport is not None:
             self.transport.close()
 
+        if self.save_state_loop is not None:
```

Comment:
This can just be `if self.save_state_loop:`

Reply:
I did that so that it was consistent to the previous ones. Also, isn't "is not None" the more "idiomatic" way to do it?
Let me know if I should update the previous ones too!

Reply:
I think the idiomatic way would be to initialize `self.save_state_loop = None` in the constructor, and then to use just `if self.save_state_loop` since it will either be a `Future` or `None`.

Reply:
Perfect, thanks. I've just pushed it, and also changed the check for "refresh_loop" as it will also be either a `Future` or `None`.